lernfragen.md

```diff
@@ -160,3 +176,19 @@ HCI Lernfragen
 ##### 79. Ein gutes Hilfesystem ist eine unerlässliche Bedingung für selbstbeschreibungfähige Systeme.
 
 ##### 80. Körperbehinderungen verursachen erhöhte Transformationsdistanzen in den unteren Ebenen des 6- Ebenen-Modells.
+
+#####______________________________________________________
+##### 81. Hintergrundbilder sollten grundsätzlich ausblendbar sein.
+Richtig.
+
+##### 82. Je genauer ein GUI-Prototyp ausgeführt ist, desto besser kann man ihn in der Evaluation einsetzen
```

Comment:
Genau. In dem Link steht halt dass ein genauerer Prototyp "...to be able to examine usability questions in detail and make strong conclusions about how behavior will relate to use of the final product..."